.github/workflows/nuget.yml

```diff
@@ -34,13 +34,13 @@ jobs:
       run: |
         dotnet pack ./neo.sln \
         --configuration Release \
-        --output ./sbin \
-        --verbosity normal \                        # Normal verbosity level
-        -p:VersionPrefix=${{ env.APP_VERSION }}     # Set the version prefix from tag_name
+        --output ./bin/packages \
```

Comment:
Does the path need to be modified?

Reply:
More consistent with the rest of the `yml` files

Reply:
Which yml files?
I did not understand, the paths are different, right?

Reply:
> Which yml files?

All the ones I been making. `release.yml`, `main.yml`, `docker.yml`, Also `C#` solution

> I did not understand, the paths are different, right?

They all fall under `bin` that will be the place you they will be stored under for all builds. So it isn't some random location a developer picked.

Reply:
I'm with @vncoelho this is necessary?

Reply:
Yes the comments, On the command-line will break the workflow and have the step `fail`.